src/react/components/shell/skipButton.tsx

```diff
@@ -11,14 +11,18 @@ export class SkipButton extends React.Component<ISkipButtonProps> {
 
     public render() {
         return (
-            <div className="skip-button">
+            <div className="skip-button" tabIndex={1}>
                 <a href="#" onClick={this.skipToId}>{this.props.children}</a>
             </div>);
     }
 
     private skipToId = (event: React.MouseEvent<HTMLAnchorElement>) => {
         event.preventDefault();
-        const element = document.getElementById(this.props.skipTo);
+
+        const collection = document.getElementsByClassName(this.props.skipTo);
+        const id = collection[0].id;
+        const element = document.getElementById(id);
```

Comment:
why not element = collection.length > 0 ? collection[0] : null?

Reply:
collection's type is HTMLCollection which cannot call focus function

Reply:
if we can just let element = collection[0], then element.foucus() doesn't exist

Reply:
convert it to element: collection[0] as HtmlElement